lib/Models/GeoJsonCatalogItem.js

```diff
@@ -448,12 +441,15 @@ function loadGeoJson(geoJsonItem) {
         markerSize : defaultValue(parseMarkerSize(style['marker-size']), 20),
         markerSymbol: style['marker-symbol'], // and undefined if none
         markerColor : defaultColor(style['marker-color'], pointPalette),
-        strokeWidth : defaultValue(style['stroke-width'], 2),
+        strokeWidth : defaultValue(style['stroke-width'], 4),
```

Comment:
This is also used for the outline of points and for polylines.  Do we want those to be width 4 too?  Also, polyline width > 1 doesn't work on Windows anyway, so I'm not sure how much value there is in bumping it up to 4.